### DISCUSSION TITLE
HELP PLEASE

### DISCUSSION THREAD

**Discussion by funkofunatic** [community/community#29836] [Other Feature Feedback, Questions, & Ideas]
So I am new to the whole bot world and I am a avid funko pop collector and just wanted to know what i need to do and how i do it but i want to get the fright night tickets for funkoville coming up in october in NYC. I know they will be on sale soon and will be being sold on eventbrite website so wanted to see what I could do to make sure I get two tickets for my wife and myself for our very first funko event? I see their is bots out there but like I said this is my first time dealing with this so I am beyond confused any help would be greatly appreciated it and i am even willing to pay someone if they want to get them for me using our names or show me how to get them. I will make it worth your time. Thank you in advance everyone!!!

**Comment by queenofcorgis** [comment]
Hi @funkofanatic we’re glad you’re here 😀,

Thanks for sharing your request with the community. Unfortunately, we currently do not allow self-promotion, solicitation, or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members. We'll be locking this post from here.
